templates/corporate/billing/upgrade.html

```diff
@@ -6,6 +6,20 @@
 {% endblock %}
 
 {% block portico_content %}
+
+{% if is_demo_organization %}
+<div class="register-account flex full-page">
+    <div class="center-block new-style">
+        <div class="white-box demo-organization-warning">
+            Demo organizations cannot be directly upgraded to a paid plan. Please start by
+            <a href="/help/demo-organizations#convert-a-demo-organization-to-a-permanent-organization">
+            converting your demo organization
+            </a>
+            to a permanent organization.
+        </div>
+    </div>
+</div>
+{% else %}
```

Comment:
This probably works, but it might be better to just have Python code code just render a shared `portico_error` type page, so we don't clutter this HTML with a case that isn't actually possible.